src/services/AnalysisService.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+type CommentMoodPayload = {
+  comment: string;
+}
+
+class AnalysisService {
+  private url: string;
+
+  constructor() {
+    this.url = process.env.AnalysisServerUrl!;
+  }
+
+  async getCommentMood(payload: CommentMoodPayload) {
```

Comment:
I would rather accept string, it will be easier for the caller. The object can be constructed in this wrapper around the ML API

In this case, even if the ML server changes the data format, we would need to edit this class only.